components/src/core/hero/hero.tsx

```diff
@@ -83,23 +83,9 @@ export type HeroProps = ViewProps & {
     iconBackgroundColor?: string;
 
     /**
-     * The text size to use for the ChannelValue
-     *
-     * Default: 20
+     * Props to be passed through to ChannelValue child component
      */
-    fontSize?: number;
-
-    /** Value for the ChannelValue child */
-    value?: number | string;
-
-    /** A component to render for the ChannelValue child */
-    valueIcon?: IconSource;
-
-    /** Color to use for the ChannelValue text */
-    valueColor?: string;
-
-    /** Units for the ChannelValue child */
-    units?: string;
+    channelValueProps?: ChannelValueProps;
```

Comment:
Our convention for Pass-Through props is to TitleCase them
```suggestion
    ChannelValueProps?: ChannelValueProps;
```